.github/workflows/business-ar-reminder-ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: Business AR Reminder CI
+
+on:
+  pull_request:
+    paths:
+      - "jobs/ar-reminder/**"
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./jobs
+
+jobs:
+  business-ar-reminder-ci:
+    uses: bcgov/bcregistry-sre/.github/workflows/backend-ci.yaml@main
+    with:
+      app_name: "business-ar-reminder"
+      working_directory: "./jobs"
```

Comment:
need to update the working directory as well ./jobs/ar-reminder

Reply:
Thank you for point it out, just updated